css/properties/clip-path.json

```
@@ -60,9 +60,9 @@
             "deprecated": false
           }
         },
-        "animations": {
+        "animation_supported": {
```

This is more consistent with your` is_transitionable` below.

```suggestion
        "is_animatable": {
```